shell/spy_test.go

```diff
@@ -1,14 +1,36 @@
 package shell
 
 import (
-	"fmt"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
 )
 
-func TestExecute(t *testing.T) {
-	want := fmt.Sprintf("spy.go\nspy_test.go\n")
-	got := Execute("ls")
-	if got != want {
-		t.Errorf("Expected: %s, got: %s", want, got)
+func TestParseCommand(t *testing.T) {
+	testData := []struct {
```

Comment:
I always call my slice of test cases `tcs`, and I recommend you do too—or, at least, I recommend you always use the same name, and if you have a better suggestion for what it should be, I'll switch to using yours instead!

Reply:
`tcs` / `tc` seems fitting for sure! I didn't really give my name choice much thought.